app/policies/board_policy.rb

```diff
@@ -18,41 +18,49 @@ def new?
   end
 
   def edit?
-    user_is_creator?
+    user_is_creator? || user_is_admin?
   end
 
   def create?
     true
   end
 
   def update?
-    user_is_creator?
+    user_is_creator? || user_is_admin?
   end
 
   def destroy?
     user_is_creator?
   end
 
   def continue?
-    user_is_creator? && can_continue?
+    (user_is_creator? || user_is_host?) && can_continue?
   end
 
   def create_cards?
     user_is_member?
   end
 
   def suggestions?
-    check?(:user_is_member?)
+    user_is_creator? || user_is_admin?
   end
 
   def invite?
-    check?(:user_is_member?)
+    user_is_creator? || user_is_admin?
   end
 
   def user_is_creator?
     record.creator?(user)
   end
 
+  def user_is_admin?
+    record.admin?(user)
```

Comment:
Same here, lets remove any usages of these methods from model 
Use record.memberships instead